### ISSUE TITLE
Scatter charts seem to always start fom a zero axis

### ISSUE THREAD

**Issue by lloydtabb** [malloydata/malloy#1106]
Reported on Slack.

![image](https://github.com/malloydata/malloy/assets/1093458/934cbdc3-38ce-426c-84a8-3ff4b43ca102)

**Comment by srschandan** [comment]
In your case the chart does seem disoriented and confusing. I can make sure that this behavior will only be localized to scatter charts. However, allowing the user to toggle this behavior may need a bigger discussion so I'll leave it at that.

**Comment by lloydtabb** [comment]
Yes.  I agree.  The non zero axis should only effect scatter charts. Bar
charts should start at 0
